core/domain/config_domain.py

```diff
@@ -285,6 +309,14 @@ def get_config_property_schemas(cls):
         '0FBWxCE5egOw', '670bU6d9JGBh', 'aHikhPlxYgOH', '-tMgcP1i_4au',
         'zW39GLG_BdN2', 'Xa3B_io-2WI5', '6Q6IyIDkjpYC', 'osw1m5Q3jK41'])
 
+TOPIC_IDS_FOR_CLASSROOM_PAGE = ConfigProperty(
+    'topic_ids_for_classroom_page', SET_OF_CLASSROOM_DICTS_SCHEMA,
+    'The set of topic IDs in the initial classroom page.', [{
```

Comment:
Isn't this "for each classroom page"? I'm a bit confused about whether we're dealing here with one page or multiple pages.

Reply:
Done